### PR TITLE
Fixes public API endpoints for models

### PR DESCRIPTION
**Pull request by manuelurenah**
Closes https://github.com/civitai/civitai/issues/1047

### CODE REVIEW

File: src/server/services/model.service.ts [hunk 1]
```diff
@@ -523,13 +538,21 @@ export const getModelsRaw = async ({
...
   }
 
+  const includeDetails = !!include?.includes('details');
+
   const modelQuery = Prisma.sql`
     ${queryWith}
     SELECT
       m."id",
       m."name",
+      ${includeDetails ? Prisma.raw('m."description",') : Prisma.empty}
```
Comment (lrojas94) [inline src/server/services/model.service.ts:548]: Is there any reason why we don't group this sub-query as a single thing? 
```
includeDetails ? Prisma.raw(`m.desc, m.poi, etc, etc`) : Prisma.empty
```

File: src/server/services/model.service.ts [hunk 2]
```diff
@@ -566,11 +594,17 @@ export const getModelsRaw = async ({
         SELECT
          jsonb_build_object(
            'id', mv."id",
+           'name', mv."name",
+           ${includeDetails ? Prisma.raw(`'description', mv."description",`) : Prisma.empty}
```
Comment (lrojas94) [inline src/server/services/model.service.ts:598]: Ditto

File: src/server/services/model.service.ts (hunk 1 above)
Comment (JustMaier) [reply src/server/services/model.service.ts:548]: Agreed.

File: src/server/services/model.service.ts [hunk 3]
```diff
@@ -523,13 +538,21 @@ export const getModelsRaw = async ({
     modelVersionWhere.push(Prisma.sql`cm."modelVersionId" = mv."id"`);
   }
 
+  const includeDetails = !!include?.includes('details');
+
```
Comment (JustMaier) [inline src/server/services/model.service.ts:542]: @lrojas94 What do you think about this approach 😉

**Comment by BlafKing** [comment]
Looks like there is still a small issue with the `sort` parameter after this merge: #1074

**Pull request merged**